### PR TITLE
Order collections by most recent, fix bug with create override

### PR DESCRIPTION
**Pull request by cameronblandford**

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def save(self, *args, **kwargs):
                 self.settings = resources.meeting_json
             elif self.collection_type == 'dataset':
                 self.settings = resources.dataset_json
-        super(Collection, self).save(args, kwargs)
+        super(Collection, self).save(*args, **kwargs)
```
Comment (NyanHelsing) [inline api/models.py]: i think it can just be `super().save` too

Comment (cameronblandford) [reply api/models.py]: I'll test it

File: api/views.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ class CollectionList(generics.ListCreateAPIView):
     permission_classes = (drf_permissions.IsAuthenticatedOrReadOnly,)
 
     def get_queryset(self):
-        queryset = Collection.objects.all()
+        queryset = Collection.objects.all().order_by('-date_created')
```
Comment (NyanHelsing) [inline api/views.py:95]: I'm thinking it might be good to sort by num of items in the collection, and by most recently added to as well.

Comment (cameronblandford) [reply api/views.py:95]: Should that be part of this PR or would it be good to make a separate one for that?

File: api/models.py [hunk 3]
```diff
@@ -37,7 +37,7 @@ def save(self, *args, **kwargs):
                 self.settings = resources.meeting_json
             elif self.collection_type == 'dataset':
                 self.settings = resources.dataset_json
-        super(Collection, self).save(args, kwargs)
+        super().save(*args, **kwargs)
```
Comment (cameronblandford) [inline api/models.py:40]: You were right @birdbrained, totally worked

**Pull request merged**